### ISSUE TITLE
docs: zh/*.md some file first dir is "名字"

### ISSUE THREAD

**Issue by liangliang4ward** [apache/apisix#6477]
### Improved or typo fixed.

like https://apisix.apache.org/zh/docs/apisix/plugins/key-auth
![image](https://user-images.githubusercontent.com/3895272/156115171-497ac496-d3bc-43f4-bb7a-f81c13c529f2.png)

Would it be better to  "简介"？or "描述"

### Solution

名称->简介

**Comment by spacewander** [comment]
The same section could be "Description" in https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/forward-auth.md or "Introduction" in https://github.com/apache/apisix/blob/master/docs/en/latest/plugins/limit-req.md.

We need a consistent type for the doc.

**Comment by kwanhur** [comment]
`Description` maybe the better.

When we fetch GNU tool's usage with `man`, `Description` is in common use.

If ok, I can fix them.

**Comment by liangliang4ward** [comment]
1. Maybe we can remove  inner dir
2. Unified directory name，include zh/*.md & en/*.md

**Comment by spacewander** [comment]
> `Description` maybe the better.
> 
> When we fetch GNU tool's usage with `man`, `Description` is in common use.
> 
> If ok, I can fix them.

I prefer to have a full discussion before rushing to a fix.

**Comment by spacewander** [comment]
CC @juzhiyuan

**Comment by juzhiyuan** [comment]
Hi,

After checking most plugins' docs (Chinese), I prefer using `Description` in English, and `描述` in Chinese 🤗 Let's think `Plugin Description` and `插件描述`.

**Comment by juzhiyuan** [comment]
I don't choose `简介` (short description) is because this section contains what's the plugin is? what're features it have? Sometimes, it's hard to describe it in one sentence.

**Comment by kwanhur** [comment]
Another viewpoint, will these plugins' docs can be converted into mandoc in the future?
If can, we can retrieve plugin's help by `man apisix-plugin-xxx` after installed apisix.

**Comment by juzhiyuan** [comment]
> Another viewpoint, will these plugins' docs can be converted into mandoc in the future? If can, we can retrieve plugin's help by `man apisix-plugin-xxx` after installed apisix.

Why not create a new issue to discuss this? 🤗

**Comment by juzhiyuan** [comment]
If there has no feedback for this issue, then let's use Description/描述 in our docs.

**Comment by liangliang4ward** [comment]
> If there has no feedback for this issue, then let's use Description/描述 in our docs.

agree

**Comment by kwanhur** [comment]
Please assign to me, I’ll work on it.


Kwanhur Huang
TL;DR

**Comment by kwanhur** [comment]
After have a look, first title `Name` `Introduction` or empty will be changed to `Description`, `名字` `简介` `插件简介` `定义` will be changed to `描述` in docs of `docs/en/latest/plugins/` and `docs/zh/latest/plugins`. 

Any other opinions? cc @spacewander @juzhiyuan

**Comment by juzhiyuan** [comment]
> After have a look, first title `Name` `Introduction` or empty will be changed to `Description`, `名字` `简介` `插件简介` `定义` will be changed to `描述` in docs of `docs/en/latest/plugins/` and `docs/zh/latest/plugins`.
> 
> Any other opinions? cc @spacewander @juzhiyuan

LGTM
